.typos.toml

```diff
@@ -1,38 +1,54 @@
 [default]
-extend-ignore-re = ["[a-zA-Z0-9]{44}", "[a-zA-Z0-9]{9}"]
+extend-ignore-identifiers-re = [
+  # Short git hashes
+  "^[a-f0-9]{7}$",
+  # Digests
+  "^[a-zA-Z0-9]{43}$",
+  # Identifiers
+  "^[a-zA-Z0-9]{44}$",
+  # Base64
+  "^[a-zA-Z0-9_]{47}$",
+  # Base64
+  "^[a-zA-Z0-9]{49}$",
```

Comment:
Any reason this one doesn't contain _ like the previous one?

Reply:
Tbh I only guessed that they are base64, but I think only one actually is. No idea what the other one is 😆